manifests/mount.pp

```diff
@@ -11,7 +11,9 @@
   $cvmfs_use_geoapi = undef,
   $cvmfs_repo_list = true,
   $cmvfs_mount_rw = undef,
-  $cvmfs_follow_redirects = undef
+  $cvmfs_follow_redirects = undef,
+  $mount_options = 'defaults,_netdev',
```

Comment:
@jblomer do you have some suggestions from experience for better mount options.

I've gone fore *defaults,_netdev* here which work.

The _netdev option instructs the fstab to mount unit generator to set sensible dependencies in the unit file to mount cvmfs after the network is up.

Should there be any other options in the default configuration?

I can make it different for SLC6 and CC7 easily if needs be.